omnibus/omnibus-test.sh

```diff
@@ -34,5 +32,4 @@ echo "Running verification for $product"
 export GEM_HOME=/SHOULD_NOT_EXIST
 export GEM_PATH=/SHOULD_NOT_EXIST
 
-export PATH="$PATH:/usr/local/bin"
 inspec version
```

Comment:
Do we need to pass the last exit code here as well ?

Reply:
no bash tests will handle failed exit codes properly. it's only in powershell that we have to explicitly test for failed exit codes.